layouts/partials/nav.html

```diff
@@ -37,7 +37,32 @@
                         {{ end }}
                     {{ end }}
                 {{ end }}
+                <li class="nav-item">
+                    <button type="button" id="refresh-trigger" class="btn btn-primary" data-toggle="modal" data-target="#refreshModal">
+                        Refresh 
```

Comment:
Perhaps discuss the "Refresh" label with the team. I wouldn't intuitively click a "Refresh" button to clear my data.

Reply:
Agreed, I'll bring it up the next time we meet.